deepchem/feat/material_featurizers/lcnn_featurizer.py

```diff
@@ -0,0 +1,664 @@
+import numpy as np
+from deepchem.feat import Featurizer
+from collections import defaultdict
+from pymatgen import Element, Structure, Molecule, Lattice
+from pymatgen.symmetry.analyzer import PointGroupAnalyzer
+import networkx as nx
+import networkx.algorithms.isomorphism as iso
+from scipy.spatial.distance import cdist, pdist, squareform
```

Comment:
By importing these at the top level, you make pymatgen, networkx, and scipy into hard dependencies.  You can move the imports into only the functions that use them?